3rdparty/aoclutils/opencl.cc

```diff
@@ -0,0 +1,555 @@
+// Copyright (C) 2013-2018 Altera Corporation, San Jose, California, USA. All rights reserved.
```

Comment:
@tqchen will this copyright work with Apache licensing rules? this will end up in 3rd party